crmsh/ui_cluster.py

```diff
@@ -637,6 +634,8 @@ def do_run(self, context, cmd, *nodes):
         opts = parallax.Options()
         opts.ssh_options = ['StrictHostKeyChecking=no']
         opts.askpass = utils.check_ssh_passwd_need(hosts)
+        if hasattr(opts, 'ssh_key'):
+            opts.ssh_key = SSH_KEY_CRMSH
```

Comment:
For now, there do exist some piece of code in crmsh using parallax directly, instead of using crmsh.parallax
That seems duplicated.
But I suggest we replace them with crmsh.parallax in other PR, not in this one, since validation those functional work was time cost